### DISCUSSION TITLE
I didn't find the eventaggregator and region

### DISCUSSION THREAD

**Discussion by mimimiaomiao1** [AvaloniaUI/Avalonia#8169] [Q&A] [answered]
Support now?

**Comment by maxkatz6** [comment, marked as answer]
Are you asking for prism support?
https://github.com/AvaloniaCommunity/Prism.Avalonia

**Reply by mimimiaomiao1** [reply]
My God, that's it, thank you Because I have used prism before, I need to use eventaggregulator and region
